node/network/src/peer.rs

```diff
@@ -39,8 +40,12 @@ const POISONED_LOCK_ERR: &str = "The lock was poisoned.";
 pub struct Handshake {
     /// Protocol version.
     pub version: u32,
-    /// Sender's peer information.
-    pub info: PeerInfo,
+    /// Sender's peer id.
+    pub peer_id: PeerId,
```

Comment:
Why do we need to replace `info: PeerInfo` with three separate fields that represent exactly the same information?